app/src/main/java/com/course_project_autotrace/RegisterNewVehicle.java

```diff
@@ -55,7 +55,7 @@ public void onDataChange(@NonNull DataSnapshot dataSnapshot) {
                     }
 
                     if (car != null) {
-                        referenceToUsers.child(userID).child(Objects.requireNonNull(car.getKey())).setValue(car.getValue()).addOnSuccessListener(aVoid -> {
+                        referenceToUsers.child(userID).child("Cars").child(Objects.requireNonNull(car.getKey())).setValue(car.getValue()).addOnSuccessListener(aVoid -> {
```

Comment:
Edits reference in the registration screen to comply with the updated database sorting system